### ISSUE TITLE
DllNotFoundExceptioin user32

### ISSUE THREAD

**Issue by cyc1ingsir** [Lswbanban/BlueBrick#5]
Hi,

this is quite a nice software. Thankyou.
I tried to get it running under Linux (opensuse Tumbleweed and Leap 15.2) and installed the `mono-complete`* package but when I try to start it via `mono BlueBrick.exe&`  I get the attached error.
![Error](https://user-images.githubusercontent.com/2297037/104621619-992bdc80-5690-11eb-9ec5-5e19948e51c8.png)

Unfortunately I don't know anything about C# and Mono to debug any further.
I managed to get it running via Wine though but were unable to change the configuration persistently. But's not really a problem.

*
```
Name           : mono-complete
Version        : 6.12.0-1.1
```

**Comment by Lswbanban** [comment]
TL;DR: Please try "mono32 BlueBrick.exe&"

The current mono interpreter does not support user32 anymore. Running with "mono32" will normally fix your problem.

It may still work with BlueBrick v1.8.2, however, I think it doesn't work anymore with version 1.9.x. I tried to run it under mono and under windows, but mono exit without any error, so I don't know what's wrong. Since now you cannot find a mono develop pre-compiled version for Windows, and since the source version doesn't compile for me, I couldn't debug BlueBrick under mono. I would have to find a pc under linux to debug it, which I don't have now.

Any help would be very appreciated, if you have Monodevelop and want to make BluerBrick run under mono, please don't hesitate to contribute.
